api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -114,4 +115,24 @@ public void sendMessage_shouldSendMessage() throws Exception {
 		}
 	}
 	
+	@Test
+	public void sendMessage_shouldSaveTheSentMessage() throws MessageException {
+		Message message = ms.createMessage("recipient@example.com", "sender@example.com", "subject", "content");
+		message.setId(123);
+		
+		try {
+			ms.sendMessage(message);
+			SentMessage sentMessage = Context.getSentMessageService().getSentMessage(123);
+			
+			Assert.assertNotNull(sentMessage);
+			Assert.assertEquals(123, (int)sentMessage.getMessageId());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://www.codacy.com/app/openmrs/openmrs-core/file/2318749994/issues/source?bid=3204099&fileBranchId=3204129#l128)